### PR TITLE
Added an example how to animate a Html.Styled node from elm-css

### PR DESCRIPTION
**Pull request by rolojf**
I couldn't figure out a way to make a useful helper function to go into  the `Animated` module as you suggested. Sorry.

I thing the `Animated.custom` helper function already defined by you, is enough already.

My very small contrib. is just to give an example about how to use this package with elm-css .

Regards,

### CODE REVIEW

File: examples/src/Utils/Animated.elm [hunk 1]
```diff
@@ -93,3 +96,28 @@ animatedTypedSvg node animation attributes children =
...
+    (List (Htmls.Attribute msg) -> List (Htmls.Html msg) -> Htmls.Html msg)
+    -> Animation
+    -> List (Htmls.Attribute msg)
+    -> List (Htmls.Html msg)
+    -> Htmls.Html msg
+animatedStyledNode nodeToAnimate animation attributes children =
+    Animated.custom
+        (\className stylesheet ->
+            nodeToAnimate
+                (Attr.class className :: attributes)
```
Comment (andrewMacmurray) [inline examples/src/Utils/Animated.elm:115]: The type signatures are a bit hard to guess but the way I've converted these to library helpers is to replace any UI specific functions like `Attr.class` or `Htmls.text` with `options.class` and `options.text`

So you'd end up with something like:

```elm
animatedStyledNode options nodeToAnimate animation attributes children =
    custom
        (\className stylesheet ->
            nodeToAnimate
                (options.class className :: attributes)
                (options.node "style" [] [ options.text stylesheet ] :: children)
        )
        animation
```

I arrived at this:

```elm
type alias ElmCssOptions attr node =
    { class : String -> attr
    , node : String -> List attr -> List node -> node
    , text : String -> node
    }


styled : ElmCssOptions attr node -> (List attr -> List node -> node) -> Animation -> List attr -> List node -> node
styled options nodeToAnimate animation attributes children =
    custom
        (\className stylesheet ->
            nodeToAnimate
                (options.class className :: attributes)
                (options.node "style" [] [ options.text stylesheet ] :: children)
        )
        animation

```

**Comment by andrewMacmurray** [comment]
@rolojf thanks for your input!

I've left a comment about adding it as a helper to the library, let me know if that makes sense or not. I think it would be nice for others to be able to re-use what you've done more easily.

Just one other thing, could you remove the committed changes to `package-lock.json`? The node dependencies haven't changed in this MR

**Comment by andrewMacmurray** [comment]
If it's too much work though I'm happy to make the changes myself, just let me know 👍

**Comment by rolojf** [comment]
Andrew, please make the changes yourself.  Thanks for the chance to contribute just a little.  But I've been very busy and for the following weeks  will be the same.

Regards,

**Comment by andrewMacmurray** [comment]
No worries, I’ll make some changes when I get a moment - mind if I ask for your review when it’s ready? (and obviously only if you have time)

**Pull request closed without merging**